### ISSUE TITLE
No audio [Kubuntu 20.04.1 LTS]

### ISSUE THREAD

**Issue by Rhyagelle** [cross-platform/audible-for-linux#4]
Hello,

I've installed the snap for this for Kubuntu, and it all seems to open and work fine in the regard that I can connect my account and whatnot, but playing the audiobooks, there is no sound. Absolutely no sound. 

Did I miss a prerequisite, or is this a known bug right now?

Thanks!

EDIT

I solved it. Short version: snap permissions were wrong. Long: go here: https://github.com/cross-platform/audible-for-linux/issues/4#issuecomment-695115321

**Comment by Rhyagelle** [comment]
Anyone? I would really love to be able to listen to my Audible books without having to resort to the online web player, which has no volume etc control.

**Comment by MarcusTomlinson** [comment]
Could you please run the app from the terminal like so: `audible-for-linux`, and share the terminal output here.

Note however that this is just the web player packaged into a snap, other than it being self-contained, you won't benefit from any extra functionality beyond what the web player provides.

**Comment by Rhyagelle** [comment]
Running the application from the terminal (Kubuntu) doesn't create an output for me. I tried playing an audiobook to see if it would create one, but it didn't Closing out of the app didn't seem to do it either. Hopefully this image uploads:

![image](https://user-images.githubusercontent.com/56734035/93225389-2a6d5f80-f740-11ea-9cce-ef477f6401b1.png)

(The reason I need this is so I can independently control volume etc as it is a separate app, leaving me to also control the browser's volume without impacting the audiobook)

**Comment by Rhyagelle** [comment]
So, I think I found the issue. Kubuntu was blocking the application itself. I didn't even know Linux could do this, at least without being set up by the user, and I've been using it for over a decade. Apparently though, it is common for snap packages. I went into the Discover store (the application store for Kubuntu, for those reading who may not know) and scrolled down to "Permissions for Audible", and adjusted it. Then I restarted the PC and audio was picking up for it. 

Hopefully it helps someone in the same position. I'm off to listen to Audible! :)
